README.md

```diff
@@ -2,44 +2,80 @@
 
 ## Summary
 
-Use graph API to add a Conversations/Become a member link to community pages on gcxchange based on user membership to group.
+- spfx-teamslink is a SPFX Application Customizer Extension that add a Conversations/Become a member link to community pages on gcxchange based on user membership to group.
 
-**_You need to update the hubSiteIds property with comma seperated GUIDs for your valid hubs. You can do so in the serve.json file or from the tenant wide exstensions list when deployed._**
+- You need to update the hubSiteIds property with comma seperated GUIDs for your valid hubs. You can do so in the serve.json file or from the tenant wide exstensions list when deployed.
 
-**_You need to update the teamslinkId in TeamsLinkApplicationCustomizer.ts to avoid duplication_**
+- You need to update the teamslinkId in TeamsLinkApplicationCustomizer.ts to avoid duplication.
 
```

Comment:
To add: It also hide the teams icon next to the title.

Reply:
Do you want me to add this point in the summary?

I didn't get what you mean?
According to the members' status the extension adding the button/link either  _**Access MS Teams**_ or Join Group on MS Team
![image](https://github.com/gcxchange-gcechange/spfx-teamslink/assets/95497896/f1e3f9d1-f419-496f-a632-e46cb710d63c)

![image](https://github.com/gcxchange-gcechange/spfx-teamslink/assets/95497896/a07996ad-d5c0-4e8a-8a00-981833a395f3)